src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partnerType = person.sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )}
+      {person.isMarried
+        ? (
+          <p className="Person__partner">
+            {`${person.partnerName} is my ${partnerType}`}
+          </p>
+        )
+        : (
+          <p className="Person__partner">I am not married</p>
+        )}
```

Comment:
```suggestion
      <p className="Person__partner">
        {person.isMarried
          ? `${person.partnerName} is my ${partnerType}`
          : 'I am not married'
        }
      </p>
```

<img width="1076" alt="image" src="https://github.com/mate-academy/react_person/assets/84296041/bd7950ea-d347-4e8b-983e-36be7dfbc313">